CMakeLists.txt

```diff
@@ -1610,9 +1615,7 @@ if (IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "centos")
   else()
     message(FATAL "Unsupported CentOS major version: ${IRODS_LINUX_DISTRIBUTION_VERSION_MAJOR}")
   endif()
-elseif (IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "almalinux")
-  set(CPACK_RPM_${IRODS_PACKAGE_COMPONENT_ORACLE_NAME}_PACKAGE_REQUIRES "${IRODS_PACKAGE_DEPENDENCIES_STRING}, irods-server = ${CPACK_RPM_PACKAGE_VERSION}-${CPACK_RPM_PACKAGE_RELEASE}, unixODBC")
-elseif (IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "opensuse")
+elseif (IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "almalinux" OR IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "rocky" OR IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "opensuse")
```

Comment:
suggestion...

`elseif (IRODS_LINUX_DISTRIBUTION_NAME IN_LIST "almalinux" "rocky" "opensuse")`

Reply:
looks like that might not be legal... dang.

https://cmake.org/cmake/help/latest/command/if.html#existence-checks

Reply:
right.